app/src/main/res/values/colors.xml

```diff
@@ -56,6 +56,11 @@
   <color name="black_54">#8A000000</color>
   <color name="whiteLight">#F9F9F9</color>
   <color name="red">#FF0000</color>
+  <!--  CONFETTI COLORS -->
```

Comment:
Just to check, have we confirmed that these colors work well against our typical backgrounds with respect to accessibility?

Reply:
All except the yellow pass the WCAG AA accessibility for UI / object components (I checked [here](https://webaim.org/resources/contrastchecker/)). Do we want to change the yellow color or are the other three enough?

Reply:
Since the confetti is decorative, it does not have to meet any contrast ratio requirements. This is one of the contrast ratio [exceptions](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html): _text or images of text that are part of an inactive user interface component, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement_.

Reply:
Thanks for the clarification! I don't think this fully qualifies as being pure decoration, though, as it does provide functionality by celebrating progress and encouraging learners to continue with lessons / explorations.
Could I propose adding an additional color that is accessible so that 4 of the 5 will be fully accessible? This wouldn't affect the amount of confetti shown but would enable more visibility for some -- let me know what you thinks!

Reply:
I would say that the "Correct" toast functions more as a way to celebrate progress than confetti does, but it's only available for questions and not shown at the end of a lesson. There _are_ visuals cues at the end of the lesson to signify its completion when there are buttons to navigate to the next chapter, so I still believe that the confetti is only meant to be visually stimulating rather than informative. 

However I am opened to seeing how other colors fare! How much would they variate from our color palette though?

Reply:
That makes sense -- the confetti might not be informative but I do think it serves a function, although it's visual stimulation might be its only source of functionality.

I took a purple color and a pink color that we use to show new profile placeholder images. It's a bit farther from our usual color palette but since confetti is meant to stand out and these colors are still present in the app it seemed fairly reasonable to use (I like the pink). Let me know your thoughts on the rotating circles as well.

**Pink**
![banner confetti pink](https://user-images.githubusercontent.com/40325757/105420191-c9c1c800-5bf4-11eb-8d44-2c2042d5f4d5.gif)

**Purple**
![banner confetti purple](https://user-images.githubusercontent.com/40325757/105420213-d47c5d00-5bf4-11eb-8c48-65ee1d3e8b60.gif)

Reply:
Thanks Jacque! I think for the pink, it needs to be saturated a bit more to fit the color theme the other colors have, maybe something like #F382FF? For the purple, I think it's looks too similar to the blue color to tell the difference between the two. Overall though I would prefer having the three primary colors (red, yellow, blue) since the background, progress bar, and correct toast make the screen vastly green. WDYT?

I also like the rotation since it does give that real-life confetti feel!

Reply:
I agree about the purple. The [mocks](https://xd.adobe.com/view/0687f00c-695b-437a-79a6-688e7f4f7552-70b6/screen/fafed02b-1f15-4bd1-9c54-8921abb796b0/specs/) have an evergreen color so then I think it would make sense to replace the green with the pink -- that way we get 3 of 4 accessible colors. Would a slightly darker shade like #EC42FF work?
![banner confetti pink](https://user-images.githubusercontent.com/40325757/105439356-904f8380-5c19-11eb-870f-271571cd613f.gif)

Reply:
Thank Jacque! The pink color still looks off to me because it doesn't fit with the other colors. Maybe white can work since it is a neutral color, WDYT? Also it might be worth finding a color or two with this [color generator](https://coolors.co/227c9d-ffcb77-fef9ef-fe6d73)!

Reply:
White wouldn't be accessible either, I'll play around with some options that might look better but let me know if you have any ideas as well!

Reply:
Decided to stick with red, yellow, and blue as three colors that contrasted well with each other and still stood out from the background. The red and blue are both accessible

Reply:
Thanks! This seems resolved now--feel free to reopen if there are additional discussion points.